### DISCUSSION TITLE
access point settings not working?

### DISCUSSION THREAD

**Discussion by roastyduck** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#794] [Q&A]
trying to set mine up however the guide kind of made sense, under access point on ssid ive put my usual internet ssid and the password i use to connect to the ssid, the module turns its ap off but checking my wifi settings on my router it doesnt show as a connected device and it also doesnt connect with http://layzspa.local. any ideas what i may be doing wrong? tried 5.8g and 2.4g networks and its the same on both. its annoying as ive got to reset flash on visual studio rebuild and upload every time i try to get the ap to turn back on

**Comment by visualapproach** [comment]
The layzspa.local may not work with all routers. If you have a 6-wire pump you can see the IP in the display at startup. Use that instead.

**Reply by roastyduck** [reply]
Got it working eventually, unsure why but I connected to my WiFi extender and it worked after that. Weird as the extender was further away
